platform/src/main/java/org/hillview/sketches/ExactCountSketch.java

```diff
@@ -0,0 +1,80 @@
+package org.hillview.sketches;
```

Comment:
you may want to install the copyright plugin for intellij, it will do copyrights automatically for you once you configure it